Wed Nov 15 11:08:51 2023       
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 530.30.02              Driver Version: 530.30.02    CUDA Version: 12.1     |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                  Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf            Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  NVIDIA A40                      On | 00000000:65:00.0 Off |                    0 |
|  0%   26C    P8               27W / 300W|      0MiB / 46068MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
                                                                                         
+---------------------------------------------------------------------------------------+
| Processes:                                                                            |
|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
|        ID   ID                                                             Usage      |
|=======================================================================================|
|  No running processes found                                                           |
+---------------------------------------------------------------------------------------+
********************************************************
steps: 1366
episodes: 10
mean 100 episode reward: 2.8
% time spent exploring: 98
********************************************************
Traceback (most recent call last):
  File "/home/fs01/dje4001/anaconda3/envs/StressDQN/lib/python3.7/site-packages/numpy/lib/npyio.py", line 1434, in savetxt
    v = format % tuple(row) + newline
TypeError: %d format: a number is required, not NoneType

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "train_atari.py", line 167, in <module>
    delimiter=',', fmt='[%1.3f, %d, %d, %d, %b]')
  File "<__array_function__ internals>", line 6, in savetxt
  File "/home/fs01/dje4001/anaconda3/envs/StressDQN/lib/python3.7/site-packages/numpy/lib/npyio.py", line 1438, in savetxt
    % (str(X.dtype), format))
TypeError: Mismatch between array dtype ('object') and format specifier ('[%1.3f, %d, %d, %d, %b]')
